tests/v1/test_list_customers.py

```diff
@@ -0,0 +1,138 @@
+import unittest
+from fastapi.testclient import TestClient
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from main import app
+from api.utils.auth import hash_password, create_access_token
+from api.db.database import Base, get_db
+from api.v1.models.user import User
+from datetime import timedelta
+
+# Database configuration
+test_db_name = '' # put your test db name
+test_db_pw = '' # put your test db pw
+SQLALCHEMY_DATABASE_URL = f"postgresql+psycopg2://postgres:{test_db_pw}@localhost:5432/{test_db_name}"
+engine = create_engine(SQLALCHEMY_DATABASE_URL)
+TestingSessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+def override_get_db():
+    db = TestingSessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+app.dependency_overrides[get_db] = override_get_db
+
+Base.metadata.create_all(bind=engine)
```

Comment:
Check how to mock a database connection. Also you guys were told to use `pytest`

Reply:
I have used pytest and mock data.